test/createStoreTests.ts

```diff
@@ -13,7 +13,7 @@ describe('createStore', () => {
         let store = createStore('testStore', initialState)();
 
         // Assert
-        expect(store).toBe(initialState);
-        expect(getRootStore().get('testStore')).toBe(initialState);
+        expect(store).toEqual(initialState);
```

Comment:
I use `toEqual` instead of `toBe` since v3 changed the default behavior from changing the object in place to cloning.

more: https://github.com/mobxjs/mobx/blob/master/CHANGELOG.md#observableplainobject-will-no-longer-enhance-objects-but-clone-instead